src/collective/cover/tests/test_upgrades.py

```diff
@@ -524,8 +524,6 @@ def test_registrations(self):
         self.assertGreaterEqual(int(version), int(self.to_version))
         self.assertEqual(self._how_many_upgrades_to_do(), 4)
 
-    # FIXME: https://github.com/collective/collective.cover/issues/633
```

Comment:
@rodfersou do not remove this or test will fail in Plone 5; only remove the FIXME comment line.